src/components/searchbox/SearchBox.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+type SearchBoxProps = {
+  searchChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+const SearchBox = ({ searchChange }: SearchBoxProps) => {
+  return (
+    <>
+      <StyledSearchBox
+        type="search"
+        placeholder="Search members"
+        onChange={searchChange}
+      />
+    </>
+
+  );
+};
+
+const StyledSearchBox = styled.input`
```

Comment:
SearchBox input field could be wider on a bigger screen or positioned to a side. What do you think @NikoLenCZ  & @VeveCambor?

![image](https://github.com/user-attachments/assets/6ba9fd46-bd65-4ea1-bbee-8daab2a0de7a)

Reply:
Adjusted on media query from 992px viewport to min-width 20rem
(generally better use relative units before px)